firebase-app-distribution/src/main/java/com/google/firebase/app/distribution/FirebaseAppDistribution.java

```diff
@@ -80,7 +80,7 @@
   }
 
   /** Constructor for FirebaseAppDistribution */
-  public FirebaseAppDistribution(
+  FirebaseAppDistribution(
```

Comment:
This constructor takes in a lifecycle notifier preventing that class from being package private. Is there some reason these constructors needed to be public?

Reply:
I don't think so. External classes should be calling the `getInstance` function, not the constructor